azure-pipelines-devtest.yml

```diff
@@ -32,16 +32,16 @@ pool: Default
 
 resources:
   repositories:
-  # Azure DevOps repository
-  - repository: tilavarauspalvelu-pipelines
-    type: git
-    # Azure DevOps project/repository
-    name: tilavarauspalvelu/tilavarauspalvelu-pipelines
+    # Azure DevOps repository
+    - repository: tilavarauspalvelu-pipelines
+      type: git
+      # Azure DevOps project/repository
+      name: tilavarauspalvelu/tilavarauspalvelu-pipelines
 
 extends:
   # Filename in Azure DevOps Repository (note possible -ui or -api)
   # Django example: azure-pipelines-PROJECTNAME-api-release.yml
   # Drupal example: azure-pipelines-drupal-release.yml
   template: azure-pipelines-tilavarauspalvelu-core-devtest.yml@tilavarauspalvelu-pipelines
   #parameters:
-    #imagetag: ${{ parameters.imagetag }}
+  #imagetag: ${{ parameters.imagetag }}
```

Comment:
we remove this and above lines while we're at it. Commented code - smells like some leftovers from the setting up process.

Reply:
Done! And looks like my vscode decided to make some formatting as well 🤦

Reply:
Okay, better now 👍